src/uct/api/uct.h

```diff
@@ -459,6 +462,17 @@ struct uct_iface_attr {
             } rndv;                      /**< Attributes related to rendezvous protocol */
         } tag;                           /**< Attributes for TAG operations */
 
+        struct {
+            size_t           max_iov;    /**< Maximal @a iovcnt parameter in
+                                              @ref uct_ep_mem_reg */
+            size_t           max_dim;    /**< Maximal dimension for a strided memory
+                                              layout with @ref uct_iov_t */
+            size_t           max_short;  /**< Maximal @a iovcnt parameter in
```

Comment:
seems like need to expose performance estimations for UMR overhead/latency from UCT, since max_short does not give an idea on what the latency actually is. Can't make the decision of pack/unpack vs. UMR, for example.